tests/compilation_tests/database_test.py

```diff
@@ -169,7 +169,7 @@ def test_update_data(self):
                     self.assertEqual(ans, database.get_data(name), name)
 
     def test_phase_as_value(self):
-        robot_state_phase = Robot_State(xpos= 0, ypos =0, heading =0, epsilon =0, max_velocity =0, radius =0)
+        robot_state_phase = Robot_State(xpos= 0, ypos =0, heading =0, epsilon =0, max_velocity =0, radius =0, phase = Phase.SETUP)
```

Comment:
are the params needed if they're the default value?

Reply:
good point, only the ones that differ from default are needed! i think the ones that are default here are xpos, ypos , heading so ill remove those